osbs/build/build_request.py

```diff
@@ -136,6 +137,20 @@ def inner_template(self):
         return self._inner_template
 
     @property
+    def customize_conf(self):
+        if self._customize_conf is None:
+            path = os.path.join(self.build_json_store, DEFAULT_CUSTOMIZE_CONF)
+            logger.debug("loading inner template from path %s", path)
+            try:
+                with open(path, "r") as fp:
+                    self._customize_conf= json.load(fp)
+            except IOError:
+                # File not found, which is perfectly fine. Just leave it as None
```

Comment:
If the file is not found, it'll try to read it from disk every time this property is accessed. Maybe set it to `{}` ?